src/router/routes/index.js

```diff
@@ -1,125 +1,9 @@
-      beforeEnter: (to, from, next) => {
-        if (from.path === "/") next({ name: "customer-dashboard" })
-        else next()
-      }
-    },
-    {
-      path: "emr",
-      name: "customer-emr",
-      meta: { pageHeader: "My EMR" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/Emr")
-    },
-    {
-      path: "emr/upload",
-      name: "customer-emr-create",
-      meta: { pageHeader: "Upload EMR" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/Emr/Create")
-    },
-    {
-      path: "emr/edit/:id?",
-      name: "customer-emr-edit",
-      meta: { pageHeader: "Edit EMR", parent: "customer-emr" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/Emr/Edit")
-    },
-    {
-      path: "emr/details/:id?",
-      name: "customer-emr-details",
-      meta: { pageHeader: "Details", parent: "customer-emr" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/Emr/Details")
-    },
-    {
-      path: "my-test/:page?",
-      name: "my-test",
-      meta: { pageHeader: "My Test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/MyTest")
-    },
-    {
-      path: "order-history-detail",
-      name: "order-history-detail",
-      meta: {
-        pageHeader: "Order History Detail",
-        parent: "customer-test"
-      },
-      component: () => import( /* webpackChunkName */ "@/views/Dashboard/Customer/Home/MyTest/OrderHistoryDetail")
-    },
-    {
-      path: "test-result",
-      name: "test-result",
-      meta: { pageHeader: "Test Result", parent: "customer-test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/MyTest/TestResult")
-    },
-    {
-      path: "request-test",
-      name: "customer-request-test",
-      meta: { pageHeader: "Request Test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/RequestTest")
-    },
-    {
-      path: "request-test/service/:flag?",
-      name: "customer-request-test-service",
-      meta: { pageHeader: "Select Services", parent: "customer-request-test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/RequestTest/SelectServicePage")
-    },
-    {
-      path: "request-test/checkout/:id?",
-      name: "customer-request-test-checkout",
-      meta: { pageHeader: "Checkout", parent: "customer-request-test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/RequestTest/Checkout")
-    },
-    {
-      path: "request-test/success/:hash?",
-      name: "customer-request-test-success",
-      meta: { pageHeader: "Success", parent: "customer-request-test" },
-      component: () => import(/* webpackChunkName */ "@/views/Dashboard/Customer/Home/RequestTest/Success")
-    },
+const indexRoutes = [
+  ...CustomerRoutes,
+  ...GeneticAnalystRoutes
```

Comment:
This is the customer UI, not the Genetic Analyst UI. Why are the Genetic Analyst pages here? It should be in the `debio-frontend` project instead.

If this dashboard is for the genetic analyst's customers then you need to name the routes differently.

Reply:
@agustinustheo From Sarah, she decided to use the customer UI instead of lab UI, that's why the page is here. The routes also has been discussed by @bukandicki with others front end developers last Friday afternoon

Reply:
@hilyds then create a new project for genetic analyst UI.